addons/static_radios/CfgVehicles.hpp

```diff
@@ -36,7 +36,7 @@ class RscAttributeTFARStaticRadioThingy: RscControlsGroupNoScrollbars {
         };*/
         class FreqTitle: RscText {
             idc=2611803;
-            text="Frequency";
+            text="$STR_TFAR_Zeus_moduleStaticRadio_FreqTitle";
```

Comment:
If the same string is used multiple times it should only be once in the core stringtable.

Reply:
Yes, I will find out what string should be use together.
But in this case, this Frequency truly only appear here once :D 
Other "Frequency" world all appear in some sentence.